packages/story-editor/src/karma/fixture/containers/designMenu.js

```diff
@@ -39,4 +39,8 @@ export class DesignMenu extends Container {
       ToggleButton
     );
   }
+
+  get borderRadius() {
+    return this.queryByRole('textbox', { name: 'Corner radius' });
```

Comment:
Nit: It's `Corner Radius` (with capital `R`) in the Design Panel, should we use the same for this, too, for consistency?

Reply:
Fixed in 135bc61.